internal/pkg/service/buffer/config/config_test.go

```diff
@@ -74,6 +74,15 @@ sink:
                 # Statistics L2 in-memory cache invalidation interval. Validation rules: required,minDuration=100ms,maxDuration=5s
                 invalidationInterval: 1s
         storage:
+            volumeAssignment:
```

Comment:
**The last commit contains updated test of the Stream API configuration, it is also a snapshot of all changes.**